cli/state_test.go

```diff
@@ -134,4 +134,28 @@ func TestStatePush(t *testing.T) {
 		err := inv.Run()
 		require.NoError(t, err)
 	})
+
+	t.Run("OtherUserBuild", func(t *testing.T) {
+		t.Parallel()
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
```

Comment:
note that we need the provisionerd since the command does a new build unlike for `pull` tests